src/pages/home/report/ReportActionItem.js

```diff
@@ -192,6 +192,15 @@ function ReportActionItem(props) {
         [props.action.reportActionID, reactionListRef],
     );
 
+    const isDeletedParentAction = ReportActionsUtils.isDeletedParentAction(props.action);
```

Comment:
Lets move this to [here](https://github.com/Expensify/App/blob/1682be47d71b5a84c20c2396e833c0a015829252/src/pages/home/report/ReportActionItem.js#L156), and add a comment above it.

Reply:
@getusha 

Done. Will you check again?